src/components/Modeler.vue

```diff
@@ -947,5 +947,19 @@ $cursors: default, not-allowed;
       }
     }
   }
+
+  .vertex > circle:hover {
```

Comment:
If you replace this selector with only `.joint-marker-vertex { ...` then it still works as expected, and you could remove the extra `vertices` code from `linkConfig.js`.